docs/src/getting-started.md

```diff
@@ -1,20 +1,18 @@
 # Getting started
 
-Kani is a Rust verification tool based on _model checking_. With Kani, you can
-ensure that wide classes of problems are absent from your Rust code by writing
-_proof harnesses_, which are broadly similar to tests (especially property
-tests).
+Kani is an open-source verification tool that uses automated reasoning to analyze Rust programs.
+Kani is particularly useful for verifying unsafe code in Rust, where many of the Rust’s usual guarantees are no longer checked by the compiler.
+Some example properties you can prove with Kani include memory safety properties (e.g., null pointer dereferences, use-after-free, etc.), the absence of certain runtime errors (i.e., panics), and the absence of some types of unexpected behavior (e.g., arithmetic overflows).
+Kani can also prove custom properties provided in the form of user-specified assertions.
 
-Kani is particularly useful for verifying unsafe code in Rust, where
-many of the language's usual guarantees can no longer be checked by the
-compiler. But it's also useful for finding panics and check user-defined
-assertions in safe Rust.
+Kani uses proof harnesses to analyze programs. Proof harnesses are similar to test harnesses, especially property-based test harnesses.
 
 ## Project Status
 
 Kani is currently under active development and has not made an official release yet.
-There is support for a fair amount of the Rust language features, but not all of them.
-If you encounter issues when using Kani we encourage you to [report them to us](https://github.com/model-checking/kani/issues/new/choose).
+There is support for a fair amount of Rust language features, but not all (e.g., concurrency).
+Please see [Limitations - Rust feature support](./rust-feature-support.md) for a detailed list of supported features.
 
-Kani usually synchronizes with the main branch of Rust every two weeks, and so
-is generally up-to-date with the latest Rust language features.
+Kani usually synchronizes with the nightly release of Rust every two weeks, and so is generally up-to-date with the latest Rust language features.
+
+If you encounter issues when using Kani we encourage you to report them to us (https://github.com/model-checking/kani/issues/new/choose).
```

Comment:
```suggestion
If you encounter issues when using Kani, we encourage you to report them to us (https://github.com/model-checking/kani/issues/new/choose).
```

Reply:
Thanks for this suggestion, made me realize that the link in this sentence was not being associated with "report them to us"!